internal/http/services/prometheus/prometheus.go

```diff
@@ -19,7 +19,9 @@
 package prometheus
 
 import (
+	"fmt"
 	"net/http"
+	"reva/pkg/metrics"
```

Comment:
Need to refer to the github repo path, I guess that's why the build isn't passing

Reply:
Missed that, good catch-up @ishank011

Reply:
@ishank011 How to resolve this? Exclude prometheus.go modifications from this pull request, and make that a separate pull request after this one is merged?

Reply:
No, just change it to `github.com/cs3org/reva/pkg/metrics`. Since the remote origin points to the repo itself, it'll resolve it itself.